akka-docs/rst/java/code/docs/pattern/SchedulerPatternTest.java

```diff
@@ -148,20 +149,20 @@ public void testSchedule(final JavaTestKit probe, Props props,
         system.eventStream().publish(new Mute(filter));
 
         final ActorRef actor = system.actorOf(props);
-        new Within(startDuration) {
-          protected void run() {
-            probe.expectMsgEquals("tick");
-            probe.expectMsgEquals("tick");
-            probe.expectMsgEquals("tick");
-          }
-        };
+        within(startDuration, () -> {
+          probe.expectMsgEquals("tick");
+          probe.expectMsgEquals("tick");
+          probe.expectMsgEquals("tick");
+          return null;
```

Comment:
yeah, on one hand it can be useful to be able to return something from the `within` block but in most cases it will not be used (and `void` would be better)